ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -200,6 +244,35 @@ device_classes: !ZenPackSpec
                                 dpName: volume_free
                                 lineType: AREA
                                 rpn: "size,-,size,/,-100,*"
+                    Physical Volume IO Stats:
+                        units: per sec
+                        miny: 0
+                        maxy: 100
+                        graphpoints:
+                            RRQM:
+                                dpName: diskstats_numReadsMerged
+                                lineType: AREA
```

Comment:
You shouldn't typically plot many areas on the same graph. It's too much visual noise. Regular lines are almost always better unless you're plotting a single datapoint that has some kind of volume property.